pkg/terminal/command.go

```diff
@@ -1073,6 +1074,18 @@ func printcontextNoState(t *Term) {
 	printcontext(t, state)
 }
 
+func (c *Commands) rebuild(t *Term, ctx callContext, args string) error {
+	if ctx.Prefix == revPrefix {
+		return c.rewind(t, ctx, args)
+	}
+	defer t.onStop()
+	discarded, err := t.client.Restart(true)
+	if len(discarded) > 0 {
+		fmt.Printf("Some of the breakpoints could not be recovered")
```

Comment:
nit: no need for capitalization. Also, maybe rephrase to `not all breakpoints could be restored`.